src/Symfony/Component/Finder/Tests/FinderTest.php

```diff
@@ -852,4 +853,15 @@ public function testMultipleLocationsWithSubDirectories()
         $this->assertIterator($expected, $finder);
         $this->assertIteratorInForeach($expected, $finder);
     }
+
+    public function testShellCommand()
+    {
+        $cmd = Command::create()->add('--force');
+        $this->assertSame('--force', $cmd->join());
```

Comment:
It's just a sanity check before checking problem case. `Command` class is not tested anywhere directly, so I thought it would be a good idea.